app/static/src/services/staffService.js

```diff
@@ -0,0 +1,24 @@
+import ServerConnector from "./serverConnector";
+let instance = null;
+export default class StaffService extends ServerConnector{
+    constructor(){
```

Comment:
if u do this in ServerConnector
 constructor(path) {
this.path = path;
}

then here u can remove whole constructor